src/test/ui/lint/suggestions.rs

```diff
@@ -14,8 +14,6 @@
 #![feature(no_debug)]
 
 #[no_mangle] static SHENZHOU: usize = 1;
-//~^ WARN static is marked #[no_mangle]
-//~| HELP try making it public
```

Comment:
Are these statics referenced somewhere? If not, we should remove them from the test altogether, I think.